README.md

```diff
@@ -28,6 +28,7 @@ You will need the following to build the latest Dynamo on Windows:
 - [Microsoft Visual Studio 2019](https://visualstudio.microsoft.com/downloads/) (any edition)
 - Microsoft .NET Framework 4.8 (included with Visual Studio 2019)
 - [GitHub for Windows](https://windows.github.com/)
+- For writing/runnning Dynamo tests within visual studio -[NUnit Test Adapter 2](https://marketplace.visualstudio.com/items?itemName=NUnitDevelopers.NUnitTestAdapter)
```

Comment:
Please write it like this: `Visual Studio`. Also, I would argue it is used for running, you can still write them in Visual Studio without it and run it with the console for example.